docs/_future/concepts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Slack developer beta
+lang: en
+slug: concepts
+order: 0
+layout: future
+permalink: /future/concepts
+---
+
+# Slack developer beta concepts <span class="label-beta">BETA</span>
+
+<div class="section-content">
```

Comment:
Since the `Slack Developer Beta` heading in the sidebar links to here, maybe we can add a sentence in this blurb to link out to the Getting Started Guide as well?

Reply:
I tried getting ride of this white space but I think it part of the grid I don't think we should play with it to much